stable/mariadb/README.md

```diff
@@ -43,122 +43,93 @@ The command removes all the Kubernetes components associated with the chart and
+| `master.persistence.size`                 | Persistent Volume Size                              | `8Gi`                                                             |
+| `master.config`                           | Config file for the MariaDB Master server           | `_default values in the values.yaml file_`                        |
+| `master.resources`                        | CPU/Memory resource requests/limits for master node | `{}`                                                              |
+| `master.livenessProbe.enabled`            | Turn on and off liveness probe (master)             | `true`                                                            |
+| `master.livenessProbe.initialDelaySeconds`| Delay before liveness probe is initiated (master)   | `120`                                                             |
+| `master.livenessProbe.periodSeconds`      | How often to perform the probe (master)             | `10`                                                              |
+| `master.livenessProbe.timeoutSeconds`     | When the probe times out (master)                   | `1`                                                               |
+| `master.livenessProbe.successThreshold`   | Minimum consecutive successes for the probe (master)| `1`                                                               |
+| `master.livenessProbe.failureThreshold`   | Minimum consecutive failures for the probe (master) | `3`                                                               |
+| `master.readinessProbe.enabled`           | Turn on and off readiness probe (master)            | `true`                                                            |
+| `master.readinessProbe.initialDelaySeconds`| Delay before readiness probe is initiated (master) | `15`                                                              |
+| `master.readinessProbe.periodSeconds`     | How often to perform the probe (master)             | `10`                                                              |
+| `master.readinessProbe.timeoutSeconds`    | When the probe times out (master)                   | `1`                                                               |
+| `master.readinessProbe.successThreshold`  | Minimum consecutive successes for the probe (master)| `1`                                                               |
+| `master.readinessProbe.failureThreshold`  | Minimum consecutive failures for the probe (master) | `3`                                                               |
+| `slave.replicas`                          | Desired number of slave replicas                    | `1`                                                               |
+| `slave.antiAffinity`                      | Slave pod anti-affinity policy                      | `soft`                                                            |
```

Comment:
Like with the master, please allow the setting of the entire affinity block.

Reply:
In terms of easiness, I think that this way is better for interacting with the command line. @prydonius @sameersbn thoughts?

Reply:
I see that the antiaffinity block we currently set depends on template values, so it could not be put directly in the Values.yaml file. Thoughts?

Reply:
ping @prydonius @timstoop